setup.py

```diff
@@ -333,7 +333,7 @@ def resolve_version():
             """.splitlines() if len(c.strip()) > 0],
         package_data={
             'chaco': [
-                'layers/data/*.svg',
+                'overlays/layers/data/*.svg',
```

Comment:
From what I can see, `chaco/layers/data/` still exists and it's included in the manifest file - but its not included in the package data in `setup.py`. Unless we need to keep the old files around, we can just get rid of them.